spec/index.spec.js

```diff
@@ -1,10 +1,10 @@
 const factory = require( "../src/index" );
-const { useFakeTimers } = require( "sinon" );
+const useFakeTimers = require( "sinon" ).useFakeTimers;
 const chai = require( "chai" );
 chai.should();
 
 describe( "aspxauth#decrypt", () => {
-	let config, token, result, clock, currentTime;
+	var config, token, result, clock, currentTime;
```

Comment:
Looks like you started to change `let`s to `var`s but we can probably change them back. There are just 3 in this file, I think.